requirements.txt

```diff
@@ -0,0 +1,3 @@
+tensorflow==1.14
+-e git://github.com/alexbrillant/Neuraxle.git@a270fe2b2f73c9350d76fcf4b6f058b764a8c8f7#egg=neuraxle
+requests
```

Comment:
@alexbrillant Let's change requests to `urllib`. Let's not import requests anymore. See how I use urllib here: 
https://www.neuraxle.org/stable/rest_api_serving.html#API-Call-Example